examples/calc/ops.ts

```diff
@@ -14,15 +14,11 @@ function div(a: number, b: number): number {
   return a / b;
 }
 
-export const ops_list: { [x: string]: (a: number, b: number) => number } = {
+export const ops_list: {[x: string]: (a: number, b: number) => number} = {
```

Comment:
nit: `opsList`? Similar below, camelCase is more typical than snake_case in JS/TS.

Reply:
Good tip, updated var names everywhere now